source/streaming/geh_stream/codelists/business_process_role.py

```diff
@@ -14,12 +14,10 @@
 from enum import Enum
 
 
-class MarketEvaluationPointType(Enum):
-    ve_production = "D01"
-    analysis = "D02"
-    surplus_production_group = "D04"
-    own_production = "D09"
-    reactive_energy = "D20"
-    consumption = "E17"
-    production = "E18"
-    exchange = "E20"
+class BusinessProcessRole(Enum):
+    unknown = 0
+    energy_supplier = 1
+    grid_access_provider = 2
+    system_operator = 3
+    metered_data_responsible = 4
+    energy_agency = 5
```

Comment:
We have been informed by SME (Christian) that the business process role for GEH when receiving time series is MeteredDataAdministrator, hence please add: 

```suggestion
    energy_agency = 5
    metered_data_administrator = 6
```

Reply:
Thank you. Added.